### PR TITLE
Fix #189 - Add known hostname / username checks from malware

### LINKED ISSUE #189: Anti-Sandbox Check - Known hostnames / usernames
Some malware will stop executing if it's running on a machine with certain host names and user names, presumably as an anti-sandbox check.  It'd be awesome if al-khaser could have a check for these.

Usernames checked for by Gootkit, from [1]:
```
CurrentUser
Sandbox
```

Hostnames checked for by Gootkit, from [1]:
```
SANDBOX
7SILVIA
```

Usernames checked for by a Trickbot downloader, from [2]: 
```
Emily
HAPUBWS
Hong Lee
IT-ADMIN
Johnson
Miller
milozs
Peter Wilson
timmy
user
```

Hostnames checked for by a Trickbot downloader, from [2]: 
```
HANSPETER-PC
JOHN-PC
MUELLER-PC
WIN7-TRAPS
```

More research will need to be done to know which sandboxes these usernames and hostnames correspond to.  For the ones that we can't definitively tie back to a known sandbox, should these be included in al-khaser?  It's possible those could be the username/hostname of the malware author's test environment, for instance, and isn't technically an anti-sandbox check.

[1] https://www.sentinelone.com/blog/gootkit-banking-trojan-deep-dive-anti-analysis-features/
[2] https://www.bromium.com/deobfuscating-ostap-trickbots-javascript-downloader/

### PR DESCRIPTION
**Pull request by recvfrom**
This implements all the checks from #189.  Let me know what you think - I'm happy to implement any suggested changes.

I'm participating in Hacktoberfest 2020, and figured this would be a good issue to tackle for that.  If you are OK with this PR would you mind adding a label to it called `hacktoberfest-accepted` so it will be counted? Thank you!

https://hacktoberfest.digitalocean.com/details#details

### CODE REVIEW

File: al-khaser/AntiVM/Generic.cpp [hunk 1]
```diff
@@ -96,7 +96,266 @@ VOID known_file_names() {
...
+		}
+
+		print_results(matched, msg);
+	}
+
+	free(NetBIOSHostName);
+	free(DNSHostName);
+}
+
+static BOOL check_file_exists(TCHAR *path) {
```
Comment (LordNoteworthy) [inline al-khaser/AntiVM/Generic.cpp:271]: There is a similar function in `Utils.cpp` called `is_FileExists()`. It uses `GetFileAttributes()` which can tells you that a file exists even if you don't have access to. Which is what you probably want here.

**Review by LordNoteworthy — COMMENTED** [review]
Hello recvfrom ! Thanks a lot for your contribution. Looks mostly fine to me !

File: al-khaser/AntiVM/Generic.cpp (hunk 1 above)
Comment (recvfrom) [reply al-khaser/AntiVM/Generic.cpp]: Oh nice, thanks! I switched the code to use that helper function

**Comment by LordNoteworthy** [comment]
Thanks !

**Pull request merged**